src/importlinter/cli.py

```diff
@@ -20,7 +20,7 @@
 @click.option("--config", default=None, help="The config file to use.")
 @click.option(
     "--contract",
-    default=(),
+    default=[],
```

Comment:
Thanks for this!

I'm wary of using a list as a default argument because of the risk of the mutable instance being changed elsewhere in the code. ([Further details here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).)

Might not matter in this case, but I'm going to look into it before merging.

Reply:
It's OK, take your time! Maybe there is a better way for this.

Reply:
> Another approach would be to use the builtin function list instead because the parameter accepts a callable.

I think that's the right option! Could we change it to that?

Also, would you mind adding a line to the `CHANGELOG`

Reply:
Done